shared/src/vyxal/RegisterHelpers.scala

```diff
@@ -0,0 +1,64 @@
+package vyxal
+
+import vyxal.conversions.{*, given}
+
+import scala.collection.mutable.ArrayBuffer
+
+
+object RegisterHelpers:
+
+	val register = Context().globals.register
+
+	def clear(): Unit = register.clearAndShrink(1)
+
+	def isEmpty: Boolean = register.length == 0
+
+	def length: VNum = VNum(register.length)
+
+	def reverseRegister(): Unit = 
+		val rev = register.reverse
+		for i <- 0 to register.length - 1 do
+			register.update(i, rev(i))
```

Comment:
Nitpick: I don't know why, but creating a new ArrayBuffer just to copy from it annoys me a little. I'd prefer something like this:
```suggestion
		for i <- 0 until register.length / 2 do
			val temp = register(i)
			register(i) = register(register.length - 1 - i)
			register(register.length - 1 - i) = temp
```
This is mostly a style thing, so feel free to ignore the suggestion

---

However, if you do ignore the above suggestion, do consider this one: Use `0 until register.length` instead of `0 to register.length - 1` (they do the same thing). It's just a style thing but I think it's reasonable to prefer `until` here